### DISCUSSION TITLE
When using the cli is there a way to disable creating files with the output?

### DISCUSSION THREAD

**Discussion by bryan-gc** [openai/whisper#998] [Q&A] [answered]
I only want the output on my terminal.

**Comment by glangford** [comment, marked as answer]
There doesn't seem to be a way to disable all of the output options right now. The best way I know of to minimize file creation and keep files "out of the way" is to choose just one file type and redirect it away, for example:

`--output_format txt --output_dir /tmp`

This redirects the output of just `.txt` to /tmp. These files are automatically deleted on reboot.
(`whisper` gives an error if you specify `/dev/null` as the output directory, that would have been the obvious choice).
